ext/imap/tests/skipif.inc

```diff
@@ -2,13 +2,13 @@
 extension_loaded('imap') or die('skip imap extension not available in this build');
 
 // Change these to make tests run successfully
-$mailbox  = '{localhost/norsh}';
+$mailbox  = '{127.0.0.1:143/debug/norsh}';
 $username = 'webmaster@something.com';
 $password = 'p4ssw0rd';
 $options = OP_HALFOPEN; // this should be enough to verify server present
 $retries = 0; // don't retry connect on failure
 
-$mbox = @imap_open($mailbox, $username, $password, $options, $retries);
+$mbox = imap_open($mailbox, $username, $password, $options, $retries);
```

Comment:
Suppression needs to stay here, otherwise SKIPIF will break.

Reply:
Forgot to revert while I was debugging the connection